app/Models/AdditionalTraining.php

```diff
@@ -18,7 +18,9 @@ class AdditionalTraining extends Model
 
```

Comment:
/**
 * @see https://laravel.com/docs/10.x/eloquent-relationships#many-to-many
 */